initScripts/x86_64/macOS_10.12/README.md

```diff
@@ -0,0 +1,38 @@
+# macOS 10.12 Sierra
+
+Shippable builds on `macOS 10.12` could be run by adding macOS nodes as [custom nodes in Shippable](http://docs.shippable.com/platform/tutorial/runtime/custom-nodes/).
+
+## Prerequisites
+- [git](https://git-scm.com/download/mac) (v2.13.6)
+- [Node.js](https://nodejs.org/en/download/) (v4.8.6)
+- [Docker](https://docs.docker.com/docker-for-mac/install/) (17.09.0-ce, build afdb6d4)
+
```

Comment:
Let's also add the optional ntp dependency.

Reply:
Mention that ntp should be installed if you see that the logs are not generated in the right order.